tensorflow/tools/ci_build/install/install_deb_packages.sh

```diff
@@ -19,17 +19,22 @@ set -e
 # Install FFmpeg from an alternate repository.
 apt-get install -y software-properties-common
 add-apt-repository -y ppa:mc3man/trusty-media
+# Install CMake 3.x
+add-apt-repository -y ppa:george-edison55/cmake-3.x
```

Comment:
@vrv This ppa should be moved to Dcokerfile.cpu, Dockerfile.gpu and Dockerfile.android. As well as the ffmpeg ppa above. They are workaround for using old trusty. Not needed on new ubuntu or debian image. Debian nightly is broken ever since the ffmpeg pa has made it in!